components/hab/src/command/studio/enter.rs

```diff
@@ -114,6 +114,7 @@ mod inner {
                     let ident = PackageIdent::from_str(&format!("{}/{}",
                                                                 super::STUDIO_PACKAGE_IDENT,
                                                                 version[0]))?;
+                    let command = exec::command_from_min_pkg(ui, super::STUDIO_CMD, &ident)?
```

Comment:
This continues the trend in this code path of "spooky action at a distance".  This not only gets the command to install (which returns a PathBuf rather than a Cmd), but it also manages the installation of the studio package. 

This should be the minimum change necessary to fix the underlying issue (still testing to verify), but depending on the timing of shipping this fix in a release  I'd like to refactor this to clean up the responsibilities of these functions to make it clear how/when a studio package is installed.